### PR TITLE
Feature/leo/register

### PR DESCRIPTION
**Pull request by DiegoSanMo**
Added register page

### CODE REVIEW

File: src/pages/Landing.tsx [hunk 1]
```diff
@@ -1,36 +1,87 @@
...
+            fontWeight: "700",
+          }}
+        >
+          {t("landing-page.title")}
         </Typography>
-        <StyledImage src={landing} alt="landing" />
-    </StyledTextContainer>
+      </StyledTextContainer>
+
+      {/* Landingpage's Paragraph */}
```
Comment (DiegoSanMo) [inline src/pages/Landing.tsx:51]: Remove this line

File: src/pages/Register.tsx [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
-  )
+    <Box
+      component="section"
+      width="50vw"
+      sx={{ display: "flex", position: "absolute", top: "10%" }}
+    >
+      <PaperStyled>
+        <Paper elevation={3}>
+          <Typography variant="h3" color="initial" justify-content=" center">
+            Registrate
```
Comment (DiegoSanMo) [inline src/pages/Register.tsx:19]: Add the  useTranslation hook to include the text and add used like `const { t } = useTranslation();`

And then change the text using the react-i8next format like:
`{t("register-page.title")}`

and also add this new text on the translation.json file that is already added on the public/locales/es
Should be something like: 
```
"register-page": {
    "register": "Registrate",

  }
```

File: src/pages/Register.tsx [hunk 3]
```diff
@@ -1,5 +1,62 @@
...
+              "& .MuiTextField-root": { m: 1, width: "25ch" },
+            }}
+            noValidate
+            autoComplete="off"
+          >
+            <div>
+              <TextField
+                required
+                id="outlined-required"
+                label="Nombre"
```
Comment (DiegoSanMo) [inline src/pages/Register.tsx:33]: Same here

And then change the text using the react-i8next format like:
`{t("register-page.label-name")}`

add this new text on the translation.json file that is already added on the public/locales/es
Should be something like: 
```
"register-page": {
    "label-name": "Nombre",

  }
```

File: src/pages/Register.tsx [hunk 4]
```diff
@@ -1,5 +1,62 @@
...
+                defaultValue=""
+              />
+              <TextField
+                required
+                id="outlined-required"
+                label="Apellido"
```
Comment (DiegoSanMo) [inline src/pages/Register.tsx:39]: Same here

File: src/pages/Register.tsx [hunk 5]
```diff
@@ -1,5 +1,62 @@
...
+              />
+              <TextField
+                required
+                id="outlined-required"
+                label="E-mail"
+                defaultValue=""
+              />
+              <TextField
+                id="outlined-number"
+                label="Edad"
```
Comment (DiegoSanMo) [inline src/pages/Register.tsx:50]: Same here

File: src/pages/Landing.tsx [hunk 6]
```diff
@@ -1,36 +1,87 @@
...
+        </Typography>
+        <Stack
+          direction={{ xs: "column", sm: "row" }}
+          spacing={{ xs: 1, sm: 2, md: 4 }}
+        >
+          <ColorButton variant="contained" size="medium">
+            Sign up
+          </ColorButton>
+          <Button variant="outlined" sx={{ color: "#efe5e5" }} size="medium">
+            Login in
```
Comment (DiegoSanMo) [inline src/pages/Landing.tsx:78]: Same here

Comment (DiegoSanMo) [inline src/pages/Landing.tsx:78]: Also this one

File: src/pages/Landing.tsx (hunk 1 above)
Comment (Guerra-San) [reply src/pages/Landing.tsx:51]: Can you explain why? I want to make it sure that we both are at the same page because it is a little bit tricky.

**Review by DiegoSanMo — COMMENTED** [review]
All good

**Pull request merged**